lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts

```diff
@@ -80,6 +82,7 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
     ) {
         this.dateFormat = this.appConfig.get('dateValues.defaultDateFormat');
         this.dateLocale = this.appConfig.get('dateValues.defaultDateLocale');
+        this.dateTimeFormat = this.appConfig.get('dateValue.defaultDateTimeFormat');
```

Comment:
@eromano should we always rely that app.config has date formats or have some sort of defaults in the code?